src/main/java/no/nav/veilarbaktivitet/avtaltMedNav/AvtaltMedNavService.java

```diff
@@ -38,6 +40,8 @@ AktivitetDTO markerSomAvtaltMedNav(long aktivitetId, AvtaltMedNav avtaltMedNav)
                 .avtalt(true)
                 .forhaandsorientering(forhaandsorientering)
                 .transaksjonsType(AktivitetTransaksjonsType.AVTALT)
+                .endretAv(innloggetBruker != null ? innloggetBruker.get() : null)
```

Comment:
Jeg sa feil på møtet. Jeg har ikke tatt med dette(at det skal oppdateres i aktivitet tabellen) bare fho tabellen. Men her kan typen være NavIdent. Trenger ikke sjekke så mange ting(lengre ned i pr) fordi vi gjør tilgangskontroll som sørger for at det er en veileder og vi henter ut veileder med `authService.getInnloggetVeilederIdent()`

Reply:
Forenklet null-sjekker, siden innloggetBruker alltid er populert